week-4/implement/get-ordinal-number.test.js

```diff
@@ -2,3 +2,29 @@
 
 // continue testing and implementing getOrdinalNumber for additional cases
 // Write your tests using Jest - remember to run your tests often for continual feedback
+
+function getOrdinal(n) {
+    let ord = 'th';
+  
+    if (n % 10 == 1 && n % 100 != 11)
+    {
+      ord = 'st';
+    }
+    else if (n % 10 == 2 && n % 100 != 12)
+    {
+      ord = 'nd';
+    }
+    else if (n % 10 == 3 && n % 100 != 13)
+    {
+      ord = 'rd';
+    }
+  
+    return ord;
+  }
+  console.log(getOrdinal(11));
+  
+  
+  test ("Get Ordinals", function () {
+    expect(getOrdinal(41)).toBe('st')
```

Comment:
There's a lot of different branches that the code can take in the function. It would be really good to get a few more test cases to make sure there's good coverage!